sklearn/svm/_classes.py

```diff
@@ -1777,3 +1807,8 @@ def predict(self, X):
         """
         y = super().predict(X)
         return np.asarray(y, dtype=np.intp)
+
+    def __sklearn_tags__(self):
```

Comment:
This could be set in BaseLibSVM if possible.

Reply:
done in 705c414